packages/fast-components-styles-msft/src/text-action/index.ts

```diff
@@ -108,8 +108,6 @@ const styles: ComponentStyles<TextActionClassNameContract, DesignSystem> = (
         },
         textAction_button: {
             borderColor: "transparent",
```

Comment:
If we're moving button styling into the button, it seems like the border should move as well. Is it possible to use an outline styled button in a text area now?

Reply:
I think we want to open it up and allow any button to be displayed. Turns out the borderColor wasn't even doing anything so I just removed.